build-tools/automation/build.groovy

```diff
@@ -85,15 +133,19 @@ timestamps {
 
         stageWithTimeout('build', 6, 'HOURS', XADir, true) {    // Typically takes less than one hour except a build on a new bot to populate local caches can take several hours
             if (isPr) {
```

Comment:
I think this `if (isPR)` could be removed/the `if`/`else` "merged" because there's nothing wrong with `make prepare jenkins`.

Reply:
Fixed with commit https://github.com/xamarin/xamarin-android/pull/2767/commits/53b4d30be01bf26945c7949a2c2337d1f9a0f8e6